venues/ICLR.cc/2019/Conference/python/bidding_stage.py

```diff
@@ -27,13 +28,58 @@
     with open('../webfield/reviewerWebfieldBiddingEnabled.js','r') as f:
         reviewers = client.get_group(iclr19.REVIEWERS_ID)
         reviewers.web = f.read()
-        client.post_group(reviewers)
+        reviewers = client.post_group(reviewers)
 
     with open('../webfield/areaChairWebfieldBiddingEnabled.js','r') as f:
         area_chairs = client.get_group(iclr19.AREA_CHAIRS_ID)
         area_chairs.web = f.read()
-        client.post_group(area_chairs)
+        area_chairs = client.post_group(area_chairs)
 
     iclr19.add_bid.invitees = [iclr19.REVIEWERS_ID, iclr19.AREA_CHAIRS_ID]
     client.post_invitation(iclr19.add_bid)
 
+    # set up User Score notes
+
+    if not all(['~' in member for member in reviewers.members]):
+        print('WARNING: not all reviewers have been converted to profile IDs. Members without profiles will not have metadata created.')
+    valid_reviewer_ids = [r for r in reviewers.members if '~' in r]
+
+    if not all(['~' in member for member in area_chairs.members]):
+        print('WARNING: not all area chairs have been converted to profile IDs. Members without profiles will not have metadata created.')
+    valid_ac_ids = [r for r in area_chairs.members if '~' in r]
+
+    client.post_invitation(iclr19.scores_inv)
+    user_score_notes = {}
+    for paper in openreview.tools.iterget_notes(client, invitation=iclr19.BLIND_SUBMISSION_ID):
+        for user_id in valid_reviewer_ids + valid_ac_ids:
+            if user_id not in user_score_notes:
+                user_score_notes[user_id] = openreview.Note.from_json({
+                    'invitation': iclr19.SCORES_INV_ID,
+                    'readers': [user_id],
+                    'writers': ['ICLR.cc/2019/Conference'],
+                    'content': {
+                        'users': [user_id],
```

Comment:
why this is an array and not an string?

Reply:
ah, it was left over from the way it was designed before. I will change it

Reply:
If we use an string we can filter the notes by content.user, see my comment below.